### ISSUE TITLE
Support message events such as message.channels

### ISSUE THREAD

**Issue by nickpwhite** [TwoBitAlchemist/django-slack-events#1]
Are there any plans to support message events? Currently when I receive a message event from Slack, I get the following error.
```
Internal Server Error: /slack/events
Traceback (most recent call last):
  File "$PROJECTDIR/lib/python3.6/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "$PROJECTDIR/lib/python3.6/site-packages/django/core/handlers/base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "$PROJECTDIR/lib/python3.6/site-packages/django/core/handlers/base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "$PROJECTDIR/lib/python3.6/site-packages/django/views/generic/base.py", line 71, in view
    return self.dispatch(request, *args, **kwargs)
  File "$PROJECTDIR/lib/python3.6/site-packages/django/utils/decorators.py", line 45, in _wrapper
    return bound_method(*args, **kwargs)
  File "$PROJECTDIR/lib/python3.6/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "$PROJECTDIR/lib/python3.6/site-packages/django_slack_events/views.py", line 44, in dispatch
    return super().dispatch(*args, **kw)
  File "$PROJECTDIR/lib/python3.6/site-packages/django/views/generic/base.py", line 97, in dispatch
    return handler(request, *args, **kwargs)
  File "$PROJECTDIR/lib/python3.6/site-packages/django_slack_events/views.py", line 57, in post
    SlackEvent.send(SlackEventType[event_type.upper()],
  File "$PROJECTDIR/lib/python3.6/enum.py", line 329, in __getitem__
    return cls._member_map_[name]
KeyError: 'MESSAGE'
[25/Apr/2020 13:48:25] "POST /slack/events HTTP/1.1" 500 104081
```

Documentation for the message.channels event can be found [here](https://api.slack.com/events/message.channels).

I'm happy to take a crack at a PR for this if you're interested.

**Comment by TwoBitAlchemist** [comment]
> I'm happy to take a crack at a PR for this if you're interested.

Please! I suspect it will probably work if you add the unhandled type [here](https://github.com/TwoBitAlchemist/django-slack-events/blob/master/django_slack_events/event_types.py). The reason I'm doing that in an `Enum` is because plain strings didn't trigger the signal filtering properly; otherwise I'd rather not whitelist.

Thanks for your interest! Feel free to send the PR even if you get stuck. Would love to try to integrate support for whatever events people are using, even if I'm not.

**Comment by nickpwhite** [comment]
This PR should fix it: https://github.com/TwoBitAlchemist/django-slack-events/pull/2

**Comment by TwoBitAlchemist** [comment]
Merged your PR and uploaded v1.0.1 to PyPI with the fix. I checked the [public events documentation](https://api.slack.com/events) again and looks like it was the only one tagged for the Events API I was missing. Thanks again!
